projectile-rails.el

```diff
@@ -518,22 +532,29 @@ The bound variable is \"filename\"."
                                           "/[0-9]\\{14\\}.*_\\(${plural}\\|${singular}\\).*\\.rb$"
                                           'projectile-rails-find-migration))
 
+(defcustom projectile-rails-resource-name-re-list
+  `("/app/models/\\(?:.+/\\)?\\(.+\\)\\.rb\\'"
```

Comment:
I'm really glad I ran into this question.  I've been curious about exactly what that ending `\\'` means for ages.